src/components/frame/Extensions/layouts/partials/ColonySwitcherContent/ColonySwitcherContent.tsx

```diff
@@ -14,6 +15,34 @@ import { ColonySwitcherContentProps } from './types';
 
 const displayName = 'frame.Extensions.partials.ColonySwitcherContent';
 
+const MSG = defineMessages({
+  currentColonytitle: {
+    id: `${displayName}.title`,
+    defaultMessage: 'Current colony',
+  },
+  joinedColonyTitle: {
+    id: `${displayName}.joinedColonyTitle`,
+    defaultMessage: 'Joined colonies',
+  },
+  emptyFilteredStateTitle: {
+    id: `${displayName}.emptyFilteredStateTitle`,
+    defaultMessage: 'No results to display',
+  },
+  emptyFilteredStateSubtitle: {
+    id: `${displayName}.emptyFilteredStateSubtitle`,
+    defaultMessage:
+      "There are no Colony's that match your search. Try searching again",
```

Comment:
no colonies instead of no Colony's probably?

Reply:
I fixed that in #1503